frontend/src/main/lib/local-assets.ts

```diff
@@ -78,6 +78,29 @@ export async function openFolder(asset_id: string) {
   shell.openPath(path.join(getDownloadFolder(), stored.folderName));
 }
 
+export async function openMaya(asset_id: string) {
+  const stored = getDownloadedVersionByID(asset_id);
+  if (!stored) return;
+  
+  const myPath = path.join(getDownloadFolder(), stored.folderName)
+  const pythonPath = path.join(__dirname,"../../../dcc/maya/MayaMiddleScript.py")
```

Comment:
do we know if this works on an actual build? like are the python scripts actually included in the bundle?

Reply:
All the dcc files will be stored under the new folder `root/dcc/` so griddle should be able to locate the python scripts.